extensions/wikia/AdEngine/js/lookup/prebid/prebid.js

```diff
@@ -53,6 +53,11 @@ define('ext.wikia.adEngine.lookup.prebid', [
 			}
 
 			win.pbjs.que.push(function() {
+
+				//@TODO remove two lines below when https://github.com/prebid/Prebid.js/issues/772 is fixed and prebid is updated
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)